webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -7,11 +7,22 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
+      <base-button
+        v-if="showDeleteImageButton"
+        class="delete-image"
+        icon="close"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-20T19:50:31Z" title="Friday, December 20th 2019, 8:50:31 pm +01:00">Dec 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -7,6 +7,12 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
+      <ds-button
+        @click.prevent="deleteImage"
+        icon="close"
+        v-show="showDeleteButton != ''"
```


Why `v-show` here and not `v-if`? And why do we compare it to `''`?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-06T09:56:24Z" title="Monday, January 6th 2020, 10:56:24 am +01:00">Jan 6, 2020</time>_

---


I changed it and only check for false